src/components/TopicIcon/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { View, Image } from 'react-native';
+import { string } from 'prop-types';
+import styles from './styles';
+
+const TopicIcon = ({ uri }) => {
```

Comment:
What makes this a _**Topic**_ icon instead of a normal icon? Is it just a style difference?

Reply:
I think you can just update the Icon component to support style prop and that way you just import it inside this component instead of building a new one from scratch

Reply:
I think that maybe it's not the best solution because right now I haven't a component `Icon`, I will have to create that component and also because I use this 'TopicIcon' component in three different components so I will have the same style duplicated in different places.